bin/faucet/src/client.rs

```diff
@@ -95,7 +95,7 @@ impl FaucetClient {
             .map_err(|err| FaucetError::InternalServerError(err.to_string()))?;
 
         let note_type = if is_private_note {
-            NoteType::OffChain
+            NoteType::Private
         } else {
             NoteType::Public
         };
```

Comment:
There is now a 3rd option `NoteType::Encrypted` -- should this already be handled here somehow? I suspect not as there is no way to submit this yet I think?